pkg/validate/metadata_validate.go

```diff
@@ -0,0 +1,28 @@
+package validate
+
+import "fmt"
+
+func ValidateDefaultChannel(metabundle *MetaBundle) error {
+	valid := false
+
+	defaultChannel := metabundle.AddonMeta.DefaultChannel
+	for _, channel := range metabundle.AddonMeta.Channels {
+		if channel.Name == defaultChannel {
+			valid = true
+		}
+	}
+
+	if !valid {
+		return fmt.Errorf("could not find defaultChannel in channels")
+	}
+	return nil
+}
+
+func GetAllMetaValidators() []Validator {
```

Comment:
would move to `validate.go` and rename to `GetAllValidators()` as we have a single global validator concept.